requirements.txt

```diff
@@ -3,6 +3,7 @@ aiosignal==1.3.1
 asgiref==3.7.2
 async-timeout==4.0.2
 attrs==23.1.0
+avidtools==0.1.1.1
```

Comment:
Since the reporting function is hardcoded right now and doesn't use `avidtools`, this dependency can be removed